src/Bicep.Core/Semantics/Namespaces/SystemNamespaceType.cs

```diff
@@ -112,6 +112,27 @@ maxLength switch
                 .WithVariableParameter("arg", LanguageConstants.Array, minimumCount: 1, "The array for concatenation")
                 .Build();
 
+            yield return new FunctionOverloadBuilder("cidrSubnet")
+                .WithReturnType(LanguageConstants.String)
+                .WithGenericDescription("Returns the specified subnet of a CIDR network.")
+                .WithRequiredParameter("network", LanguageConstants.String, "The string containing an IP network (CIDR format)")
+                .WithRequiredParameter("cidr", LanguageConstants.Int, "The CIDR to be used to subnet the current IPNetwork")
+                .WithRequiredParameter("subnetIndex", LanguageConstants.Int, "The index to select the subnet from the range of CIDR subnets")
+                .Build();
+
+            yield return new FunctionOverloadBuilder("cidrHost")
+                .WithReturnType(LanguageConstants.String)
+                .WithGenericDescription("Calculates the ith IP address on a network.")
```

Comment:
>ith [](http://example.com/codeflow?start=56&length=3)

Since `i` in `ith` is not mentioned anywhere, should we change to "Calculates the IP address of the specified host on a network."?